### ISSUE TITLE
Build project without including the moment node packages and include original moment dist files to use the moment plugin

### ISSUE THREAD

**Issue by jgujgu** [fullcalendar/fullcalendar#5206]
### Bug Reports
**TLDR: Do `moment` and `moment-timezone` need to be included in node packages in order to use `@fullcalendar/moment-timezone`, or can `@fullcalendar/moment-timezone` really use a globally included `moment-timezone`?**

At the end of the [Moment Plugins](https://fullcalendar.io/docs/moment-plugins#moment-timezone) section, the documentation says this:
> If you are using <script> tags and browser globals, you must ensure the original non-plugin moment.js and moment-timezone.js dist files from the Moment site is included on your page first.

This implies a globally included `moment-timezone` will work adequately. Is this really the case? The background is that I have `moment` and `moment-timezone` included as part of my project's Rails asset pipeline for legacy reasons. Moreover, all my other Vue code currently uses that `moment` successfully. In my attempts to declutter repeated dependencies, I hope to get rid of the npm installed version of `moment` and `moment-timezone`, which is purely for the Full Calendar plugin. However, when `webpack/webpacker` tries to compile when `moment/moment-timezone` are not included as node packages, it errors out:     
```
ERROR in ./node_modules/@fullcalendar/moment-timezone/main.esm.js
Module not found: Error: Can't resolve 'moment' in '/node_modules/@fullcalendar/moment-timezone'

ERROR in ./node_modules/@fullcalendar/moment-timezone/main.esm.js
Module not found: Error: Can't resolve 'moment-timezone/builds/moment-timezone-with-data' in '/node_modules/@fullcalendar/moment-timezone'
```

It's too hard for me to build a sandbox of my setup exactly, but I have built a more contrived example to try to illustrate the point. The first sandbox has the node packages, the second just tries to include moment with script tags.
**Successful Sandbox:**
https://codesandbox.io/s/fullcalendar-vue-2pfbr
**Unsuccessful Sandbox:**
https://codesandbox.io/s/fullcalendar-vue-769dp

Judging from the main Full Calendar repo, I suppose I could see why a global `moment` might not work.
https://github.com/fullcalendar/fullcalendar/blob/master/packages/moment-timezone/src/main.ts

But your confirmation or disconfirmation would really help here.

**Comment by acerix** [comment]
As far as I know, you still need the node packages to build it, but in theory could use the dist moment to run it since that is how the pre-built version works.

**Comment by jgujgu** [comment]
@acerix Thank you. I'll list the steps we took here to make our Rails assets look like node packages. Uses npm's [externals functionality](https://webpack.js.org/configuration/externals/#root).

`config/webpack/environment.js`
```js
environment.config.merge({
   externals: {
     moment: 'moment',
     'moment-timezone/builds/moment-timezone-with-data': 'moment' // trick fullcalendar/moment-timezone
   }
 });
```
`package.json`
```js
...
    "moduleNameMapper": {
...
       "moment": "<rootDir>/vendor/assets/javascripts/moment"
     },
```
